jdk/src/share/classes/sun/nio/cs/ext/GB18030.java

```diff
@@ -12595,6 +12637,7 @@ private CoderResult encodeArrayLoop(CharBuffer src,
             int condensedKey = 0;  // expands to a four byte sequence
             int hiByte = 0, loByte = 0;
             currentState = GB18030_DOUBLE_BYTE;
+            boolean isGB18030_2000 = ExtendedCharsets.isGB18030_2000();
```

Comment:
Here, and in other places: naming these vars IS_2000 would have simplified reviewing (mechanical comparison with 11)

Reply:
I think some renaming is intentional as it's a static boolean constant in 11u, but a method and a private instance variable in 8u. I do think the `isGB18032_2000` is odd (presumably to avoid a name conflict with the method), so I will alter that to just `is_2000`.

Reply:
Went with making that `isGB18030_2000` in the end and the method `IS_2000`. The latter certainly shortens some of the `GB18030.java` changes.